server/tests/materials.test.ts

```diff
@@ -0,0 +1,640 @@
+import request from 'supertest'
+import dotenv from 'dotenv'
+import { IPayload } from '../utils/jwtUtils'
+import jwt from 'jsonwebtoken'
+
+import { Material } from '../db/models/material'
+
+dotenv.config()
+const app = require('../app')
+
+jest.mock('@google-cloud/storage', () => {
+    const createWriteStreamMock = jest.fn()
+    createWriteStreamMock.mockReturnValue({
+        on: jest.fn((event: string, callback: Function) => {
+            if (event === 'error') {
```

Comment:
I would storage string variables in constants to avoid reapeat the same in multiple places

Reply:
Ok, I will define constants to avoid duplicity of strings